test/support/log.js

```diff
@@ -0,0 +1,17 @@
+
+import moment from 'moment';
+import util from 'util';
+
+import {
+  config
+} from './index';
```

Comment:
I would like to move this file into the utils directory, but this dependency upon index needs to be removed first.